examples/convert_mri_and_trans.py

```diff
@@ -134,16 +135,20 @@
 anat_dir = t1w_bids_path.directory
 
 ###############################################################################
-# Let's have another look at our BIDS directory
+# Let's have another look at our BIDS directory and plot the written landmarks
 print_dir_tree(output_path)
 
+plot_anat_landmarks(t1w_bids_path, vmax=160)
+plt.suptitle('T1 MRI')
```

Comment:
Can this be moved into `plot_anat_landmarks`? It would be nice if the figure came with a sensible title by default

Reply:
Would also be good to briefly explain what `vmax` is doing here?